docs/reference/search/suggesters/misc.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[returning-suggesters-type]]
+=== Returning the type of the suggester
```

Comment:
At first I was wondering if we should document this parameter since it is only for internal use, but then I saw you also did this for aggregations, so I agree it might be useful for anybody who needs to parse the response.

Reply:
This has had me stuck for a day, as to why my aggs all of a sudden had a prefix to the name. Is there a way to disable?

Reply:
this is concerning, this change was backwards compatible meaning that typed_keys defaults to false, and only the java high-level REST client enables them. Can you describe what you have experienced?